src/new/connection.js

```diff
@@ -69,6 +69,7 @@ class Connection extends EventEmitter {
     } else {
       this.setState(Connection.CONNECTING);
     }
+    this.emit('is-online', value);
```

Comment:
Why do we need this event? Can't the app use the `state` event?

Reply:
Yes, it can. Makes sense